c2corg_ui/static/partials/cards/images.html

```diff
@@ -0,0 +1,16 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <img ng-src="{{cardCtrl.createImg()}}" alt="{{locale['title']}}">
```

Comment:
No, it uses a new function `app.utils.createImageUrl(filename, suffix)` so that it will download a Medium Image (MI) in this case.